src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/LimitedPartnershipSubmissionCreatedResponseDto.java

```diff
@@ -2,4 +2,6 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public record LimitedPartnershipSubmissionCreatedResponseDto(@JsonProperty("id") String id) {}
+public record LimitedPartnershipSubmissionCreatedResponseDto(
```

Comment:
this re-format was to fix a sonar code smell. It said that the JsonProperty was an unused import, but it wasn't unused, so I thought perhaps the Sonar analyser was confused due to the JsonProperty usage being inline. This seems to have resolved it now as the PR reports no code smell anymore.